packages/regenerator-runtime/runtime.js

```diff
@@ -206,9 +206,12 @@ var runtime = (function (exports) {
   // Note that simple async functions are implemented on top of
   // AsyncIterator objects; they just return a Promise for the value of
   // the final result produced by the iterator.
-  exports.async = function(innerFn, outerFn, self, tryLocsList) {
+  exports.async = function(innerFn, outerFn, self, tryLocsList, PromiseImpl) {
+    if (PromiseImpl === undefined) PromiseImpl = Promise;
```

Comment:
```suggestion
    if (PromiseImpl === void 0) PromiseImpl = Promise;
```
A little shorter, if you like.